tests/requirements-relaxed.txt

```diff
@@ -24,14 +24,16 @@ antsibull-docs==2.3.1
     # via -r tests/requirements-relaxed.in
 antsibull-docs-parser==1.0.0
     # via antsibull-docs
-async-timeout==4.0.2
+async-timeout==4.0.3
```

Comment:
It's better to make general version bumps in a separate PR so that this one would only contain things concerning the new dependency.

Reply:
fair point. I had to fiddle with the noxfile to get this. if I've done it correctly then we could add another session that allows introducing/upgrading individual dependencies.

e.g. I used this for the relaxed requirements: `nox -e "pip-compile-3.10(requirements-relaxed)" -- --upgrade-package sphinx-intl`

Reply:
Yes, this is the way.

Reply:
@webknjaz FYI that @gotmax23 added this here: https://github.com/ansible/ansible-documentation/commit/7bec1d92c6c8cf2f3e40c810f31245f1abce28e5